ads/_config.js

```diff
@@ -105,6 +105,8 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
+  adocean: {},
```

Comment:
Guidelines suggests that renderStart API implementation is optional. We have planned to implement renderStart API together with ad-resize implementation in a near future.